Content.Server/Administration/ServerAPI.cs

```diff
@@ -0,0 +1,795 @@
+                {
+                    await context.RespondAsync("Round already started", HttpStatusCode.BadRequest);
+                    _sawmill.Info("Forced round start failed: round already started");
+                    return true;
+                }
+                _taskManager.RunOnMainThread(() =>
+                {
+                    ticker.StartRound();
+                });
+                _sawmill.Info("Forced round start");
+                break;
+            case "end":
+                if (ticker.RunLevel != GameRunLevel.InRound)
+                {
+                    await context.RespondAsync("Round already ended", HttpStatusCode.BadRequest);
+                    _sawmill.Info("Forced round end failed: round is not in progress");
+                    return true;
+                }
+                _taskManager.RunOnMainThread(() =>
+                {
+                    roundEndSystem.EndRound();
+                });
+                _sawmill.Info("Forced round end");
+                break;
+            case "restart":
+                if (ticker.RunLevel != GameRunLevel.InRound)
+                {
+                    await context.RespondAsync("Round already ended", HttpStatusCode.BadRequest);
+                    _sawmill.Info("Forced round restart failed: round is not in progress");
+                    return true;
+                }
+                _taskManager.RunOnMainThread(() =>
+                {
+                    roundEndSystem.EndRound();
+                });
+                _sawmill.Info("Forced round restart");
+                break;
+            case "restartnow": // You should restart yourself NOW!!!
+                _taskManager.RunOnMainThread(() =>
+                {
+                    ticker.RestartRound();
+                });
+                _sawmill.Info("Forced instant round restart");
+                break;
+            default:
+                await context.RespondErrorAsync(HttpStatusCode.BadRequest);
+                return true;
+        }
+
+        _sawmill.Info($"Round {action} by {actor!.Name}({actor!.Guid}).");
+        await context.RespondAsync("Success", HttpStatusCode.OK);
+        return true;
+    }
+#endregion
+
+#region Fetching
+
+    /// <summary>
+    ///     Returns an array containing all presets.
+    /// </summary>
+    private async Task<bool> GetForcePresets(IStatusHandlerContext context)
+    {
+        if (context.RequestMethod != HttpMethod.Get || context.Url!.AbsolutePath != "/admin/force_presets")
+        {
+            return false;
+        }
+
+        if (!CheckAccess(context))
+        {
+            await context.RespondErrorAsync(HttpStatusCode.Unauthorized);
+            return true;
+        }
+
+        var jObject = new JsonObject();
+        var presets = new List<string>();
+        foreach (var preset in _prototypeManager.EnumeratePrototypes<GamePresetPrototype>())
+        {
+            presets.Add(preset.ID);
```

Comment:
Should probably include the preset description too so it can be communicated to admins.